$MirOS$

	Backport fix from Python 2.7 to work with newer zlib

--- Lib/test/test_zlib.py.orig	Wed Apr  9 00:26:21 2008
+++ Lib/test/test_zlib.py	Wed May 28 21:16:00 2014
@@ -69,7 +69,7 @@ class ExceptionTestCase(unittest.TestCas
 
     def test_baddecompressobj(self):
         # verify failure on building decompress object with bad params
-        self.assertRaises(ValueError, zlib.decompressobj, 0)
+        self.assertRaises(ValueError, zlib.decompressobj, -1)
 
     def test_decompressobj_badflush(self):
         # verify failure on calling decompressobj.flush with bad params
